docs/README.md

```diff
@@ -12,14 +12,23 @@ Follow the instructions below to build the docs.
 1. To build the docs, you need to install a developer environment and run `tox`:
 
    ```shell
-   python3 -m vevn .venv
+   python3 -m venv .venv
    source .venv/bin/activate
    python -m pip install -U pip tox
    tox -e docs
    ```
 
    This runs Sphinx in your shell and outputs to `docs/build/html/`.
 
+   > **Note:** Your virtual environment should use Python 3.9, not 3.10+:
```

Comment:
what is it about Python 3.10+ that is unsupported in the docs build?

Reply:
When I build with Python 3.10, I get this error:

```
ImportError: cannot import name 'Union' from 'types' (/usr/lib/python3.10/types.py)
```

According to [this Issue](https://github.com/sphinx-doc/sphinx/issues/9562) and [this PR](https://github.com/zopefoundation/zope.schema/pull/109), Python 3.10 isn't supported until Sphinx version 4.2.0.

We're limiting our Sphinx version to 3.6:

```
Sphinx<3.6
```

Reply:
that makes sense. And is there a reason we need Python 3.9 over 3.8?

Reply:
That's a good point. Can you review the new copy?

Reply:
comment looks clearer now, thanks for updating